README.md

```diff
@@ -1,12 +1,12 @@
 django-request-logging
 ==========================
 
-Plug django-request-logging into your Django project and you will have intuitive and color coded request/response payload logging, for both web requests and API requests. Supports Django 1.8+.
+Plug anx-django-request-logging (a fork of (Rhumbix django-request-logging)[https://github.com/Rhumbix/django-request-logging/]) into your Django project and you will have intuitive and color coded request/response payload logging, for both web requests and API requests. Supports Django 1.8+.
```

Comment:
Please remove the `anx-` prefix and the mention that this is a fork, this information is already available with GitHub:
![Selection_182](https://user-images.githubusercontent.com/4420927/154627778-8ef888c9-ed81-4247-ab3a-3788ea09887a.png)

Reply:
Removed.